statsmodels/sandbox/stats/multicomp.py

```diff
@@ -585,19 +594,20 @@ def runbasic(self, useranks=False):
         self.groupmeanfilter = grouprankmean[self.intlab]
         #return grouprankmean[intlab]
 
-    def groupdemean(self):
-        """groupdemean"""
+    def groupedmean(self):
```

Comment:
it's not a grouped_mean
it's group_demean, i.e. data is demeaned by groups (i.e. residual of a group indicator model)